bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/zk/ZKMetadataDriverBase.java

```diff
@@ -260,6 +260,21 @@ public synchronized LedgerManagerFactory getLedgerManagerFactory()
         return lmFactory;
     }
 
+    @SneakyThrows
```

Comment:
@SneakyThrows is usually a bad thing as it hides potential problems

we use Checked exceptions in order to tell to the developer that things can go wrong and failures should be handled

Reply:
OK, I will fixed.